src/main/java/ChatReques11.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+/**
+ * @author madhankumar
+ */
+@Setter
+@Getter
+@ToString
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ChatReques11 {
+    private String model;
+    private List<?> messages;
+    private int n;
+    private double temperature;
+
```

Comment:
Using '==' for string comparison can lead to unexpected behavior. Use '.equals()' method for string comparison.